README.md

```diff
@@ -42,6 +42,11 @@ To install the plugin, run:
   - `status <CR>` : Print the phase and conditions of CRs of a specific type, such as `cephobjectstore`, `cephfilesystem`, etc
   - `purge-osd <osd-id> [--force]` : Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
 
+- [debug](#debug-mode) `<subcommand>...`  : Debug a deployment by scaling it down and creating a debug copy. This is supported for mons and OSDs only"
```

Comment:
The rendered link doesn't look quite right how I suggested it the first time. How about this?
```suggestion
- `debug <subcommand>...`  : [Debug a deployment](#debug-mode) by scaling it down and creating a debug copy. This is supported for mons and OSDs only"
```